tests/core/test_processor.py

```diff
@@ -181,6 +181,38 @@ async def test_reminder_scheduled(
     assert t.events[-1] == ActionExecuted("action_listen")
 
 
+async def test_reminder_lock(
+    default_channel: CollectingOutputChannel, default_processor: MessageProcessor
+):
+    from io import StringIO
+
+    logger = logging.getLogger("rasa.core.lock_store")
```

Comment:
please use the caplog fixture instead (e.g. see [here](https://github.com/RasaHQ/rasa/blob/9913c1d53fd8a78b4564ebabb18a0bde9a939611/tests/core/test_tracker_stores.py#L274) on its usage)